extensions/wikia/FeedsAndPosts/FeedsAndPosts.setup.php

```diff
@@ -4,9 +4,19 @@
  * Injects discussion feed into an article page
  */
 
+$dir = dirname( __FILE__ ) . '/';
+
 // Autoload
 $wgAutoloadClasses['FeedsAndPostsHooks'] =  __DIR__ . '/FeedsAndPostsHooks.class.php';
```

Comment:
Could we update this line so it's using `$dir` as well? That way we're not doing the same thing in 2 different ways.